docs/install.md

```diff
@@ -42,7 +42,7 @@ curl -sL https://containerlab.dev/setup | sudo bash -s "install-docker"
 Containerlab can be installed using the [installation script](https://github.com/srl-labs/containerlab/blob/main/get.sh) that detects the operating system type and installs the relevant package:
 
 /// note
-Containerlab is distributed via deb/rpm packages, thus only Debian- and RHEL-like distributives can leverage package installation.  
```

Comment:
@jenslink thanks for the PR
can you bring back the two trailing spaces where they were removed (potentially by the formatter you have)? They denote Carriage Return and are used for the formatting purposes.